src/Cli.Tests/ConfigureOptionsTests.cs

```diff
@@ -188,6 +188,44 @@ public void TestAddAzureLogAnalyticsOptions()
             Assert.AreEqual("dce-endpoint-test", config.Runtime.Telemetry.AzureLogAnalytics.Auth.DceEndpoint);
         }
 
+        /// <summary>
+        /// Tests that running the "configure --file" commands on a config without file sink properties results
+        /// in a valid config being generated.
+        /// </summary>
+        [TestMethod]
+        public void TestAddFileSinkOptions()
+        {
+            // Arrange
+            _fileSystem!.AddFile(TEST_RUNTIME_CONFIG_FILE, new MockFileData(INITIAL_CONFIG));
+
+            Assert.IsTrue(_fileSystem!.File.Exists(TEST_RUNTIME_CONFIG_FILE));
+
+            // Act: Attempts to add file options
+            ConfigureOptions options = new(
+                fileSinkEnabled: CliBool.True,
+                fileSinkPath: "/custom/log/path.txt",
```

Comment:
The expected value `"/custom/log/path.txt"` tested here can be defined as a constant, so it can be reused in the Assert below. This way any change needed has to made only at one place.